### ISSUE TITLE
False positives in UNUSED_IMPORT

### ISSUE THREAD

**Issue by petertrr** [cqfn/diKTat#742]
## Describe the bug
Unused imports rule should not check `get` and `set` methods that can use brackets notation:
```kotlin
import com.example.get

val a = list[1]  // equivalent to list.get(1) using com.example.get
```

## Environment information
* diktat version: 0.4.0

**Comment by petertrr** [comment]
Another FP:
```kotlin
import js.externals.jquery.`$`

`$`("document").ready {}
```

**Comment by petertrr** [comment]
Another FP for infix function:
```kotlin
import org.akuleshov7.utils.logAndExit

fun main() {
    "Type $type is not supported yet" logAndExit 1
}
```

**Comment by petertrr** [comment]
I believe, also `invoke` method imports should not be checked.
